book/src/configuration.md

```diff
@@ -41,7 +41,7 @@ You may also specify a file to use for configuration with the `-c` or
 | `shell` | Shell to use when running external commands. | Unix: `["sh", "-c"]`<br/>Windows: `["cmd", "/C"]` |
 | `line-number` | Line number display: `absolute` simply shows each line's number, while `relative` shows the distance from the current line. When unfocused or in insert mode, `relative` will still show absolute line numbers. | `absolute` |
 | `cursorline` | Highlight all lines with a cursor. | `false` |
-| `gutters` | Gutters to display: Available are `diagnostics` and `line-numbers` and `spacer`, note that `diagnostics` also includes other features like breakpoints, 1-width padding will be inserted if gutters is non-empty | `["diagnostics", "line-numbers"]` |
+| `gutters` | Gutters to display: Available are `diagnostics` and `diff` and `line-numbers` and `spacer`, note that `diagnostics` also includes other features like breakpoints, 1-width padding will be inserted if gutters is non-empty | `[ "diff", "diagnostics", "line-numbers"]` |
```

Comment:
Trying this out and I noticed that `s/diff/git-diff`

Reply:
I am not sure what you mean with that. Could you elaborate?

Reply:
Sorry, to be more clear... it seems like the config is looking for "git-diff" instead of "diff". Not sure if you intended for it to actually be diff or git-diff.
```
❯ hx .config/helix/config.toml
Bad config: unknown variant `diff`, expected one of `diagnostics`, `line-numbers`, `spacer`, `git-diff` for key `editor.gutters` at line 20 column 1
Press <ENTER> to continue with default config
```

Reply:
Never mind, you can ignore me.  It looks like my local rebase may be funky.  I see in this PR that the enum is actually `GutterType::Diff`